docker-compose.yml

```diff
@@ -9,14 +9,12 @@ services:
     environment:
       - DB_USER="postgres"
       - DB_PASS="postgres"
-      - DB_NAME="blog2_db_1"
+      - DB_NAME="elephant_in_the_room_dev"
 
-
-  adminer:
-    image: adminer
+  pgadmin4:
+    image: fenglc/pgadmin4
     restart: always
```

Comment:
I believe restart is only for deployment, because if pgadmin4 fails it will be always restarted and that isn't something we want on development. From the docs "Always restart the container regardless of the exit status. When you specify always, the Docker daemon will try to restart the container indefinitely"

Reply:
[Fixed!](https://github.com/lambdaclass/elephant_in_the_room/pull/2/commits/6cb2ad19c2da81345d54ddd0bb970555b19b92d0)